hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/NeighborEntitiesRequestBuilder.java

```diff
@@ -85,7 +85,12 @@
             context, timeRange, this.getOutgoingEdges(neighborFields)),
         (resultSetRequest, metricRequestList, incomingEdges, outgoingEdges) ->
             new NeighborEntityRequest(
-                entityScope, resultSetRequest, metricRequestList, incomingEdges, outgoingEdges));
+                entityScope,
+                resultSetRequest,
+                metricRequestList,
+                incomingEdges,
+                outgoingEdges,
+                false));
```

Comment:
Should this be inherited, same as time range (and spaces for that matter, in my WIP PR)

Reply:
I was not sure. I don't think we want the time agnostic nature for `NeighborEntitiesRequestBuilder`. I am not sure even sure what would that mean

Reply:
hmm - I guess interactions don't support time agnostic today, which means that any neighbors described by interactions would necessarily be live in the time range, so we can assume this to be false. That does rely on the implementation of interactions, but it doesn't make sense to change that behavior without a good reason because it would be a lot of work - could you leave a comment to this effect explaining why this is hardcoded?